frontend/src/components/eMIB/EmailPreview.jsx

```diff
@@ -1,30 +1,116 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
 
 const styles = {
+  //buttons
   button: {
-    width: 180,
-    marginBottom: 10,
-    marginLeft: 20,
-    marginRight: 20,
-    textAlign: "center",
+    width: 155,
+    textAlign: "left",
     borderRadius: 4,
     padding: 6,
-    border: "2px solid #009fae"
+    border: "2px solid #00565E",
+    cursor: "pointer"
+  },
+  // 3x background colors
+  button_selected_background: {
+    backgroundColor: "#00565E"
+  },
+  button_read_background: {
+    backgroundColor: "#F5F5F5"
+  },
+  button_unread_background: {
+    backgroundColor: "white"
+  },
+  // 2x text colors
+  button_selected_text: {
+    color: "#D3FCFF"
+  },
+  button_unselected_text: {
+    color: "black"
+  },
+  //li
```

Comment:
I'm confused by some of these comments. This one doesn't feel required since it doesn't add any context outside of the code, and I'm not sure what "2x text colors" refers to.

Reply:
removed